src/Foundation.Find.Cms/Config/PropertyListBase.cs

```diff
@@ -0,0 +1,25 @@
+﻿using EPiServer.Core;
+using EPiServer.Framework.Serialization;
+using EPiServer.Framework.Serialization.Internal;
+using EPiServer.ServiceLocation;
+using Newtonsoft.Json;
+
+namespace Foundation.Find.Cms.Config
```

Comment:
Change namespace to Foundation.Find.Cms.Facets.Config

Reply:
Done